[2025-01-06T09:46:45.428+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.md_account_d manual__2025-01-06T09:46:40.712844+00:00 [queued]>
[2025-01-06T09:46:45.445+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.md_account_d manual__2025-01-06T09:46:40.712844+00:00 [queued]>
[2025-01-06T09:46:45.446+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-06T09:46:46.120+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): md_account_d> on 2025-01-06 09:46:40.712844+00:00
[2025-01-06T09:46:46.127+0000] {standard_task_runner.py:60} INFO - Started process 136 to run task
[2025-01-06T09:46:46.132+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'md_account_d', 'manual__2025-01-06T09:46:40.712844+00:00', '--job-id', '1920', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpn8neirof']
[2025-01-06T09:46:46.135+0000] {standard_task_runner.py:88} INFO - Job 1920: Subtask md_account_d
[2025-01-06T09:46:46.200+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.md_account_d manual__2025-01-06T09:46:40.712844+00:00 [running]> on host 7a6df65f772c
[2025-01-06T09:46:46.302+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abershits' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='md_account_d' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T09:46:40.712844+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T09:46:40.712844+00:00'
[2025-01-06T09:46:51.339+0000] {base.py:83} INFO - Using connection ID 'postgres_db' for task execution.
[2025-01-06T09:46:51.524+0000] {base.py:83} INFO - Using connection ID 'postgres_db' for task execution.
[2025-01-06T09:46:51.546+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-06T09:46:51.560+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=md_account_d, execution_date=20250106T094640, start_date=20250106T094645, end_date=20250106T094651
[2025-01-06T09:46:51.603+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-06T09:46:51.642+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
